rasa_sdk/knowledge_base/actions.py

```diff
@@ -253,7 +265,7 @@ async def _query_attribute(
         )
 
         slots = [
-            SlotSet(SLOT_OBJECT_TYPE, object_type),
+            SlotSet(SLOT_OBJECT_TYPE, None),
```

Comment:
Similar here, why is `object_type` replaced with `None` ?

Reply:
Same reason as for the previous question. By resetting the object_type slot throughout the conversation, a user can ask **"what is the price range of Berlin Burrito Company?"** without first asking to list the objects for an object type.

Reply:
Likewise, here, add a brief comment why the reset is required.

Reply:
Done 👍🏽

Reply:
Thanks! Added a rephrasing suggestion in both comments above.

Reply:
Thank you! Incorporated the suggested changes in both comments